opentreemap/opentreemap/settings/default_settings.py

```diff
@@ -357,7 +357,17 @@
 # STORAGE_UNITS is the unit the value will be stored/computed as,
 # if different from DISPLAY_DEFAULTS
 #
-STORAGE_UNITS = {}
+STORAGE_UNITS = {
+    'bioswale': {
+        'drainage_area': 'sq_m'
+    },
+    'rainGarden': {
+        'drainage_area': 'sq_m'
+    },
+    'greenInfrastructure': {
+        'area': 'sq_m'
+    }
+}
```

Comment:
All of our other storage units are in imperial.

Since we haven't deployed this feature to production yet, we don't have any data migration issues to worry about, so I think we should use imperial units for storage for these fields as well (and continue to leave `STORAGE_UNITS` empty)